q2_longitudinal/_utilities.py

```diff
@@ -27,28 +27,40 @@
 TEMPLATES = pkg_resources.resource_filename('q2_longitudinal', 'assets')
 
 
+def _validate_input_values(state_1, state_2):
+    if state_1 == state_2:
+        raise ValueError((
+            'You have chosen the same value for state_1 and state_2. These '
+            'parameters must be given different values.'))
+
+
 def _get_group_pairs(df, group_value, individual_id_column='SubjectID',
                      group_column='Group', state_column='time_point',
-                     state_values=['1', '2'], drop_duplicates=True):
+                     state_values=['1', '2'], replicate_handling='error'):
     results = []
     group_members = df[group_column] == group_value
     group_md = df[group_members]
     for individual_id in set(group_md[individual_id_column]):
         result = []
         for state_value in state_values:
             state_value = df[state_column].dtype.type(state_value)
-            individual_id = \
-                df[individual_id_column].dtype.type(individual_id)
+            individual_id = df[individual_id_column].dtype.type(individual_id)
             _state = df[state_column] == state_value
             _ind = df[individual_id_column] == individual_id
             individual_at_state_idx = group_md[_state & _ind].index
             if len(individual_at_state_idx) > 1:
                 print("Multiple values for {0} {1} at {2} {3} ({4})".format(
                     individual_id_column, individual_id, state_column,
                     state_value, ' '.join(map(str, individual_at_state_idx))))
-                if drop_duplicates:
+                if replicate_handling == 'error':
+                    raise ValueError((
+                        'Replicate values for individual {0} at state {1}. '
+                        'Remove replicate values from input files or set '
+                        'replicate_handling parameter to select how '
+                        'replicates are handled.'))
+                elif replicate_handling == 'drop':
                     break
-                else:
+                elif replicate_handling == 'random':
                     individual_at_state_idx = [choice(individual_at_state_idx)]
             elif len(individual_at_state_idx) == 0:
```

Comment:
Instead of ending this with `break`, could you insert `pass` instead of break, and then put `result.append(individual_at_state_idx[0])` in an `else` block?